src/cc/bcc_syms.cc

```diff
@@ -158,6 +158,18 @@ int ProcSyms::_add_module(const char *modname, uint64_t start, uint64_t end,
     auto module = Module(
         modname, check_mount_ns ? ps->mount_ns_instance_.get() : nullptr,
         &ps->symbol_option_);
+
+    // pid/maps doesn't account for file_offset of text within the ELF.
+    // It only gives the mmap offset. We need the real offset for symbol
+    // lookup.
+    if (module.type_ == ModuleType::SO) {
+      if (bcc_elf_get_text_scn_info(modname, &module.elf_so_addr_,
+                                    &module.elf_so_offset_) < 0) {
+        fprintf(stderr, "ERROR: Couldn't find .text section in %s\n", modname);
```

Comment:
Can we use `WARNING` instead of `ERROR`? we actually proceeded with `elf_so_addr_ = elf_so_offset_ = 0`, which may produce correct result.

Reply:
Sure, I'll change it to WARNING and repost. thanks